diff --git a/dist/src/index.js b/dist/src/index.js
index c26c1d2481e5567f91f0b9b32fd1b5d43fd13f37..22ea00b858cf1944c0daab0c322f5267a81bb382 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -91342,7 +91342,8 @@ See https://developers.cloudflare.com/cloudflare-one/access-controls/service-cre
       sockets,
       extensions,
       structuredLogging: this.#structuredWorkerdLogs,
-      autogates: process.env.MINIFLARE_WORKERD_AUTOGATES ? process.env.MINIFLARE_WORKERD_AUTOGATES.split(" ") : []
+      autogates: process.env.MINIFLARE_WORKERD_AUTOGATES ? process.env.MINIFLARE_WORKERD_AUTOGATES.split(" ") : [],
+      v8Flags: process.env.MINIFLARE_V8_FLAGS ? process.env.MINIFLARE_V8_FLAGS.split(" ") : []
     };
   }
   async #assembleAndUpdateConfig() {
